staging/src/k8s.io/legacy-cloud-providers/azure/azure.go

```diff
@@ -197,6 +197,12 @@ type Config struct {
 	//   "external": for external LoadBalancer
 	//   "all": for both internal and external LoadBalancer
 	PreConfiguredBackendPoolLoadBalancerTypes string `json:"preConfiguredBackendPoolLoadBalancerTypes,omitempty" yaml:"preConfiguredBackendPoolLoadBalancerTypes,omitempty"`
+	// EnableMultipleStandardLoadBalancers determines the behavior of the standard load balancer. If set to true
+	// there would be one standard load balancer per agent pool, which is similar with the behavior of the basic
+	// load balancer. Users could select the specific standard load balancer for their service by the service
+	// annotation `service.beta.kubernetes.io/azure-load-balancer-mode`, If set to false, the standard load balancer
+	// would keep the current behavior, which is one per cluster. In this case, the mode selection annotation would be ignored.
```

Comment:
s/the standard load balancer would keep the current behavior, which is one per cluster/the same standard load balancer would be shared by all services in the cluster/